xml/System.Text.Encodings.Web/TextEncoderSettings.xml

```diff
@@ -13,8 +13,8 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a filter which allows only certain Unicode code points through.</summary>
```

Comment:
which --> that
allows only certain Unicode code points through --> allows only certain Unicode code points